packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -51,6 +51,8 @@
     "SFDX: Execute Anonymous Apex with Currently Selected Text",
   "feature_previews_title": "Salesforce Feature Previews",
   "force_project_create_text": "SFDX: Create Project",
+  "force_project_create_change_set_based_text":
+    "SFDX: Create Change-set-based Project",
```

Comment:
@ruthemmanuelle - Is this the proper casing for the command?

Reply:
No. In title-case capitalization, words that would be capitalized if they weren't hyphenated should be capitalized when they follow hyphens. This should be "SFDX: Create Change-Set-Based Project".